block_my_feedback.php

```diff
@@ -132,7 +133,43 @@ public function fetch_feedback() : array {
 
         // Template data for mustache.
         $template = new stdClass();
+        $template->feedback = [];
+        $i = 0; // We grabbed up to 10 grades but only want to show up to 5 - so count the output.
         foreach ($submissions as $f) {
+            // Check if a quiz feedback should be shown.
+            if ($f->modname == 'quiz') {
+                $settings = $DB->get_record('quiz', ['id' => $f->instance]);
+                // Codes for all combinations of checked options except closed.
+                $closednotchecked = [0, 256, 4096, 4352, 65536, 65792, 69632, 69888];
```

Comment:
Are there any methods in the quiz module that could be reused instead of hardcoding the settings' values?

Reply:
Found a nice way to determine values programatically. Thanks @aydevworks .